src/Semantics/OperationName/ClientRequestOperationNameResolver.php

```diff
@@ -13,6 +13,13 @@ class ClientRequestOperationNameResolver implements ClientRequestOperationNameRe
 {
     public function getOperationName(string $method, string $url, string $peerName): string
     {
+        $hostname = parse_url($url, \PHP_URL_HOST);
+        $protocol = parse_url($url, \PHP_URL_SCHEME);
+
+        if (!empty($hostname) && !empty($protocol)) {
+            $url = sprintf('%s://%s', $protocol, $hostname);
```

Comment:
Is it mean that the operation name won't include URI at all, just ahost and a protocol?

Reply:
Yes, indeed, you're right.
It should be added as attribute using semantic conventions, notably: